### ISSUE TITLE
Paging of output in a tutorial mode

### ISSUE THREAD

**Issue by SorraTheOrc** [Azure/simdem#105]
SimDem2 does not implement the paging of output in tutorial mode, this makes it very hard to follow what it happening. That is, in SimDem1 a new heading triggered a `clear` in the terminal, making it much easier to view the outputs as they quickly scrolled passed.

This, coupled with #102 (color coded output) makes it very difficult to follow what is happening.

**Comment by lastcoolnameleft** [comment]
It used to do this, but I stripped it during some debugging.  

Here's a really simple PR for someone.  You can add `self.ui.clear()` here:
https://github.com/Azure/simdem/blob/simdem2/simdem/mode/tutorial.py#L21
